src/ruby-gem/lib/twsearch/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Twsearch
+  VERSION = "0.0.1"
```

Comment:
I don't suppose there's a nice way to pick this up from the Rust version?

Reply:
If the Rust version exposes its own version, then it is definitely possible!
The only thing that is not possible is to let the Ruby gem dynamically assume the `VERSION` constant from the `Cargo` file definition.

Reply:
Is the Rust lib code self-aware enough to expose its own version?

Reply:
> Is the Rust lib code self-aware enough to expose its own version?

It could be! That would probably be a good idea to add to the API/FFI sooner rather than later.